dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -1122,3 +1124,14 @@ CWWKD1110.incompat.with.collection.explanation=Some of the Query by Method Name
  keywords are not compatible with entity attributes that have a collection type.
 CWWKD1110.incompat.with.collection.useraction=Update the repository method \
  to avoid using the keyword that is not compatible with the collection type.
+
+CWWKD1111.unsupported.convert=CWWKD1111E: The {0} AttributeConverter \
+ implementation converts {1} values to the {2} type, which is not \
+ supported by Jakarta Data. Jakarta Data supports Java enumeration types, \
```

Comment:
suggest saying
the Jakarta Data specification
instead of just
Jakarta Data

(Applies to multiple instances, including in the explanation. You can leave the sentence that starts "The built-in..." as it is.)

Reply:
Updating as suggested